### ISSUE TITLE
HTML element <details> doesn't work

### ISSUE THREAD

**Issue by Aleksandr-Buyakevich** [tinymce/tinymce#9118]
I insert the next HTML to the editor, using the "Source code" mode. The "details" element always looks expanded and clicking by it doesn't force collapsing:
&lt;details&gt;
&lt;summary&gt;Summary&lt;/summary&gt;
&lt;p&gt;Text&lt;/p&gt;
&lt;/details&gt;

TinyMCE version: 6.4.2

P.S. before we used 4.7.4 and &lt;details&gt; worked correctly: collapse/expand its content by click

**Comment by TheSpyder** [comment]
I can't reproduce this as you describe in version 4, did you have a plugin that made this work for you?

TinyMCE 6.5 added the [accordion](https://www.tiny.cloud/docs/tinymce/6/accordion/) plugin which is designed to manage details elements, including a button to expand and collapse.

**Comment by Aleksandr-Buyakevich** [comment]
I do not use any special plugin to make "details" work, also I excluded all plugins on initialization of tinymce - "details" still works.

After, I started tests with different versions of tinymce, and got next results

tinymce_4.7.4_dev - works
tinymce_4.7.8_dev - works
tinymce_4.7.10_dev - works
tinymce_4.7.11_dev - works
tinymce_4.7.12_dev - doesn’t work
tinymce_4.7.13_dev - doesn’t work
tinymce_4.8.0_dev - doesn’t work
tinymce_4.9.11_dev - doesn’t work
tinymce_6.4.2_dev - doesn’t work

I used “dev” versions downloaded from https://www.tiny.cloud/get-tiny/self-hosted/

**Comment by TheSpyder** [comment]
Interesting. Turns out 4.7.12 is when explicit support for details was added:
https://github.com/tinymce/tinymce/blob/86935fe5787bda009586c41a9df3ec3a85ecb0e7/modules/tinymce/CHANGELOG.md#L2107-L2111

I tracked it down, and we do this because clicking _anywhere_ in the summary text toggles the details element. This is a poor user experience. If we could limit it to clicks on the arrow, we would - but browsers don't expose that in the click event.
https://github.com/tinymce/tinymce/blob/dc062e7028d2cc7226110431ebf72c94acd9bcba/modules/tinymce/src/core/main/ts/selection/DetailsElement.ts#L6-L12

The accordion plugin includes a context toolbar button to toggle expand and collapse.
